Dockerfile.template

```diff
@@ -1,9 +1,10 @@
-ARG ARCH=
-
-FROM ${ARCH}php:%PHP_BASE_IMAGE%
+FROM php:%PHP_BASE_IMAGE%
 
 LABEL maintainer="Garcia MICHEL <garcia@soamichel.fr>"
 
+ARG DOLI_VERSION
+ENV DOLI_VERSION ${DOLI_VERSION}
+
```

Comment:
DOLI_VERSION is used on docker-run script, it needs this env var setup

Reply:
Yes right. So what is the benefit of adding the ARG ?

Reply:
I was preparing my other PR https://github.com/tuxgasy/docker-dolibarr/pull/103 and I estimate it was cleaner to get through build args, to pass args to build instead of having a template. but I admit, it's not point in this PR...

I'll do cleaning on code, separating the two subjects.
I always have more than one thing into my mind ... ^_^